migrations/202401092201_add_events_id_index.go

```diff
@@ -0,0 +1,17 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+// Create a composite index to improve performance of finding the latest nostr event for an app
+var _202401092201_add_events_id_index = &gormigrate.Migration{
+	ID: "202401092201_add_events_id_index",
+	Migrate: func(tx *gorm.DB) error {
+		return tx.Exec("CREATE INDEX idx_nostr_events_app_id_and_id ON nostr_events(app_id, id)").Error
```

Comment:
`IF NOT EXISTS ` is missing here

Reply:
https://github.com/getAlby/nostr-wallet-connect/commit/9f0585676bccd6f991f68cf73be9d42e5fbd233c